include/mbedtls/md.h

```diff
@@ -63,65 +65,69 @@ typedef enum {
 #endif
 
 /**
- * Opaque struct defined in md_internal.h
+ * Opaque struct defined in md_internal.h.
  */
 typedef struct mbedtls_md_info_t mbedtls_md_info_t;
 
 /**
- * Generic message digest context.
+ * The generic message-digest context.
  */
 typedef struct {
-    /** Information about the associated message digest */
+    /** Information about the associated message digest. */
     const mbedtls_md_info_t *md_info;
 
-    /** Digest-specific context */
+    /** The digest-specific context. */
     void *md_ctx;
 
-    /** HMAC part of the context */
+    /** The HMAC part of the context. */
     void *hmac_ctx;
 } mbedtls_md_context_t;
 
 /**
- * \brief Returns the list of digests supported by the generic digest module.
+ * \brief           This function returns the list of digests supported by the 
+ *                  generic digest module.
  *
- * \return          a statically allocated array of digests, the last entry
+ * \return          A statically allocated array of digests. The last entry
```

Comment:
Suggestion: It's a pre-existing defect that the documentation doesn't provide information on the format of the identifiers it returns. We can't change the signature of `mbedtls_md_list`, but we might add a comment saying "Each entry of the returned list is an integer belonging to the message-digest enumeration mbedtls_md_type_t`.

Reply:
Added "Each entry of the returned list is an integer belonging to the message-digest enumeration #mbedtls_md_type_t" to description